### ISSUE TITLE
Backport glibc 2.31 compatibility fix from Chromium to Electron v7/v6?

### ISSUE THREAD

**Issue by ChALkeR** [electron/electron#22291]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * `7.1.12`
  * `6.1.7`
* **Operating System:**
  * Arch Linux, `Linux xps 5.5.4-arch1-1 #1 SMP PREEMPT Sat, 15 Feb 2020 00:36:29 +0000 x86_64 GNU/Linux`
  * glibc `2.31-1`

Electron 8 not affected (see below, fixed upstream). Can't reproduce the issue on 5 and 4.

### Expected Behavior

Nothing crashing, no failures reported into the terminal.

### Actual Behavior

* Devtools and/or window contents crash.
* This is printed into terminal: `../../sandbox/linux/seccomp-bpf-helpers/sigsys_handlers.cc:**CRASHING**:seccomp-bpf failure in syscall 0230`.

### How to reproduce

* Use a system with glibc 2.31 or later
* Create a clean project, add `electron@7` to deps there, v7.1.12 will get installed.
* Run `./node_modules/.bin/electron` (without an app, to see the default welcome screen)
* Toggle devtools on
* Resize window (might need several attempts, e.g. attempt to maximize/minimize)

In some apps, "resize window" isn't even needed, it just crashes by itself on launch, it's added here to show this reproducing on pure Electron.

### Screenshots
![Screenshot_20200220_014626](https://user-images.githubusercontent.com/291301/74883704-96dad880-5382-11ea-9cdb-9c9296b924a6.png)

### Additional Information

Currently, in Electron 7 (and 6) apps, windows randomly go blank (sometimes could be triggered by resize) and print:
```
../../sandbox/linux/seccomp-bpf-helpers/sigsys_handlers.cc:**CRASHING**:seccomp-bpf failure in syscall 0230
../../sandbox/linux/seccomp-bpf-helpers/sigsys_handlers.cc:**CRASHING**:seccomp-bpf failure in syscall 0230
../../sandbox/linux/seccomp-bpf-helpers/sigsys_handlers.cc:**CRASHING**:seccomp-bpf failure in syscall 0230
../../sandbox/linux/seccomp-bpf-helpers/sigsys_handlers.cc:**CRASHING**:seccomp-bpf failure in syscall 0230
``` 

This seems to be caused by Chromium bug, https://bugs.chromium.org/p/chromium/issues/detail?id=1025739, ~and could be fixed by floating https://bugs.chromium.org/p/chromium/issues/attachmentText?aid=421799 (which is a trivial 2-line change)~ upd: https://github.com/chromium/chromium/commit/54407b422a9cbf775a68c1d57603c0ecac8ce0d7 is the actually landed fix.

Electron 8 _should_ be fine.

Also see https://bugzilla.redhat.com/show_bug.cgi?id=1773289.

glibc 2.31 was recently released (2020-02-01) and is already hitting some distros (e.g. Arch Linux).

----

Electron 7 -- affected.
Electron 6 -- affected.
Electron 5 -- can't reproduce (not sure why, didn't investigate).
Electron 4 -- can't reproduce (not sure why, didn't investigate).

Electron 8 -- can't reproduce (as expected, fix landed in Chromium 80 upstream).

----
Basically, https://github.com/chromium/chromium/commit/54407b422a9cbf775a68c1d57603c0ecac8ce0d7 should be backported to v7.x and v6.x.

Note that the commit message has a mistake -- it mentions glibc 2.30, while only 2.31+ is affected, see https://bugs.chromium.org/p/chromium/issues/detail?id=1025739#c17

**Comment by ChALkeR** [comment]
Electron 6 also affected.

**Comment by ChALkeR** [comment]
I confirmed that this is fixed in 7.x ([v7.1.14](https://github.com/electron/electron/releases/tag/v7.1.14) release), and it seems that the patch also landed to 6.x ([v6.1.9](https://github.com/electron/electron/releases/tag/v6.1.9) release).

I don't think there is a reason to keep this open.

Thanks for the fix!
